doc/howtos/quickstart.rst

```diff
@@ -12,53 +12,117 @@ Quickstart
    :local:
 
 
+Conventions
+-----------
+
+.. code-block:: python
+
+ c = ics.Calendar()
+ e = ics.Event()
+ a = ics.Attendee()
 
 Importing a Calendar from a File
 --------------------------------
 
 .. code-block:: python
 
-    from ics import Calendar
-    import requests
+ from ics import Calendar
+ import requests
 
-    url = "https://urlab.be/events/urlab.ics"
-    c = Calendar(requests.get(url).text)
+ url = "https://urlab.be/events/urlab.ics"
+ c = Calendar(requests.get(url).text)
 
-    c
-    # <Calendar with 118 events and 0 todo>
-    c.events
-    # {<Event 'Visite de "Fab Bike"' begin:2016-06-21T15:00:00+00:00 end:2016-06-21T17:00:00+00:00>,
-    # <Event 'Le lundi de l'embarqué: Adventure in Espressif Non OS SDK edition' begin:2018-02-19T17:00:00+00:00 end:2018-02-19T22:00:00+00:00>,
-    #  ...}
-    e = list(c.timeline)[0]
-    "Event '{}' started {}".format(e.name, e.begin.humanize())
-    # "Event 'Workshop Git' started 2 years ago"
+ c
+ # <Calendar with 118 events and 0 todo>
+ c.events
+ # {<Event 'Visite de "Fab Bike"' begin:2016-06-21T15:00:00+00:00 end:2016-06-21T17:00:00+00:00>,
+ # <Event 'Le lundi de l'embarqué: Adventure in Espressif Non OS SDK edition' begin:2018-02-19T17:00:00+00:00 end:2018-02-19T22:00:00+00:00>,
+ #  ...}
+ e = list(c.timeline)[0]
+ "Event '{}' started {}".format(e.name, e.begin.humanize())
+ # "Event 'Workshop Git' started 2 years ago"
 
 
 Creating a new Calendar and Add Events
 --------------------------------------
 
+:class:`Calendar` objects each represent an unique RFC 5545 iCalendar. They contain :class:`Event`, :class:`Todo` and :class:`Timeline` iterators.
+
+Time and date are represented as :class:`datetime` objects.
+
+.. code-block:: python
+
+ from datetime import datetime
+ from ics import Calendar, Event
+ c = Calendar()
+ e = Event()
+ e.summary = "My cool event"
+ e.description = "A meaningful description"
+ e.begin = datetime.fromisoformat('2022-06-06T12:05:23+02:00')
```

Comment:
Wouldn't be easier for the reader to use something like `e.begin = datetime.datetime(2022, 6, 6, 12, 5, 23, tzinfo=something)` so that the reader does not need to know about ISO 8601 ?

Reply:
included both possibilities
https://github.com/ics-py/ics-py/pull/340/commits/10a32d0cd15d6d82f9f84544c8bdb2b9e2c49284